main/zoe/guide/README.md

```diff
@@ -78,7 +78,7 @@ the swap contract behaves badly, we will both get a refund, and at
 best, we'll get what we each wanted.
 
 Let's look at the basic `publicSwap` contract ([full text of
```

Comment:
We renamed `publicSwap` to `atomicSwap` and moved all of the core zoe files under src/, so let's link to https://github.com/Agoric/agoric-sdk/blob/master/packages/zoe/src/contracts/atomicSwap.js